ush/templates/FV3LAM_wflow.xml

```diff
@@ -434,15 +597,14 @@ MODULES_RUN_TASK_FP script.
 ************************************************************************
 ************************************************************************
 -->
-{%- if do_ensemble %}
-  <metatask name="run_ensemble">
-
-    <var name="{{ ensmem_indx_name }}">
-{%- for m in range(1, num_ens_members+1) -%}
-  {%- set fmtstr=" %0"~ndigits_ensmem_names~"d" -%}
-  {{- fmtstr%m -}}
-{%- endfor %} </var>
-{%- endif %}
+ {%- if do_ensemble %}
+ <metatask name="run_ensemble">
+   <var name="{{ ensmem_indx_name }}">
+ {%- for m in range(1, num_ens_members+1) -%}
+ {%- set fmtstr=" %0"~ndigits_ensmem_names~"d" -%}
+ {{- fmtstr%m -}}
+ {%- endfor %} </var>
+ {%- endif %}
```

Comment:
I think the indent format at the left side (the original version) is more readable for this part